src/core/client.ts

```diff
@@ -244,6 +244,7 @@ class ApiClient<R extends RawApi> {
     ) => {
         const payload = p ?? {};
         debug(`Calling ${method}`);
+        if (signal !== undefined) validateSignal(method, payload, signal);
```

Comment:
Null check could be added too

Reply:
It would be a type error to pass `null` as a signal, so why not make it a runtime error, too?

Reply:
Wait no, I highlighted the wrong line